frontend/src/libs/components/card/card.tsx

```diff
@@ -20,37 +21,44 @@ const Card: React.FC<Properties> = ({
   onClick,
   isActive = false,
   iconName,
+  linkTo,
   iconColor = 'currentColor',
 }) => {
   const hasNoImageOrIcon = !imageUrl && !iconName;
   const hasImage = Boolean(imageUrl);
   const hasIcon = Boolean(iconName);
 
   return (
-    <button
-      className={getValidClassNames(
-        styles['item'],
-        isActive && styles['selected'],
-        hasNoImageOrIcon && styles['no-image'],
-      )}
-      onClick={onClick}
-    >
-      {!hasNoImageOrIcon && (
-        <div className={styles['image-container']}>
-          {hasImage && (
-            <div className={styles['image-placeholder']}>
-              <img src={imageUrl} alt="not found" className={styles['image']} />
-            </div>
-          )}
-          {hasIcon && (
-            <div className={styles['icon-background']}>
-              <Icon name={iconName as IconName} color={iconColor} />
-            </div>
-          )}
-        </div>
-      )}
-      <div className={styles['title']}>{title}</div>
-    </button>
+    <Link to={linkTo ?? '/'}>
```

Comment:
why don't just wrap a card into a link in a parent component?

Reply:
Yeah, I rewrote it similar to your component.